### PR TITLE
Update zh_cn.lang

### PR DESCRIPTION
**Pull request by Snapshotlight**
edit the translation file so that it would be more readable and be closer to the meaning.

### CODE REVIEW

File: src/main/resources/assets/abyssalcraft/lang/zh_cn.lang [hunk 1]
```diff
@@ -869,20 +869,20 @@ key.staff_mode.desc=改变守门者法杖的模式
...
+death.attack.coralium=%1$s死于珊瑚瘟疫
 death.attack.coralium.player=%1$s在试图逃离%2$s时屈服于珊瑚瘟疫之下
-death.attack.dread=%1$s由于恐惧瘟疫而死亡
+death.attack.dread=%1$s死于恐惧瘟疫
 death.attack.dread.player=%1$s在试图逃离%2$s时屈服于查加罗特的拳头之下
 death.attack.antimatter=%1$s被解离成分子
 death.attack.antimatter.player=%1$s在试图逃离%2$s时被解离成分子
-death.attack.shadow=%1$s被黑暗侵蚀了
-death.attack.shadow.player=%1$s在试图逃离%2$s时被黑暗侵蚀
+death.attack.shadow=%1$s被黑暗充满
```
Comment (Determancer) [inline src/main/resources/assets/abyssalcraft/lang/zh_cn.lang:878]: "consume"不应翻译为“充满”，其词义为"destroy completely"，"eat immoderately"。应翻译为“被黑暗吞噬”。
879行同理。

"consume" is not "filled with", it means "destroy completely", "eat immoderately". the right translation should be "eaten by darkness".
Also works for Line 879.

File: src/main/resources/assets/abyssalcraft/lang/zh_cn.lang [hunk 2]
```diff
@@ -1493,10 +1493,10 @@ subtitle.abyssalcraft.abyssalzombie.hit=深渊僵尸：受伤
 subtitle.abyssalcraft.abyssalzombie.death=深渊僵尸：死亡
 subtitle.abyssalcraft.antiplayer.hurt=反物质玩家：受伤
 subtitle.abyssalcraft.dreadguard.barf=恐惧守卫：呕吐
-subtitle.abyssalcraft.jzahar.blast=BOOM
+subtitle.abyssalcraft.jzahar.blast=爆炸
 subtitle.abyssalcraft.jzahar.shouts=扎哈尔试图 Fus Ro Dah
-subtitle.abyssalcraft.jzahar.earthquake=扎哈尔试图重排板块
-subtitle.abyssalcraft.jzahar.implosion=扎哈尔试图用自己的意志替代重力
-subtitle.abyssalcraft.jzahar.black_hole=扎哈尔试图掩盖声音花园的歌声
+subtitle.abyssalcraft.jzahar.earthquake=扎哈尔：试图重排板块
```
Comment (Determancer) [inline src/main/resources/assets/abyssalcraft/lang/zh_cn.lang:1498]: 1497行格式不一致？

Line 1497. Is the form inconsistent?

**Comment by Determancer** [comment]
翻译中有些小问题，已批注，还请更改。
另外您尚未签署CLA协议，还请留意。

File: src/main/resources/assets/abyssalcraft/lang/zh_cn.lang (hunk 2 above)
Comment (Snapshotlight) [reply src/main/resources/assets/abyssalcraft/lang/zh_cn.lang:1498]: 我没看出啥差别……

**Comment by Determancer** [comment]
如果按照 扎哈尔：...
的格式，那么应该是 扎哈尔：试图Fus Ro Dah

**Comment by Determancer** [comment]
还有一处问题尚未修改：somewhat不是“不知为何”，而是“有一点”的意思。请参考词典。

**Comment by Determancer** [comment]
advancements.abyssalcraft.abyssaln.description=合成深渊之书。扎哈尔有几分很欣赏你。
“有几分”和“很”重复了吧......

**Comment by Snapshotlight** [comment]
删除的时候没改干净，光注意“不知为何”了

**Comment by Shinoow** [comment]
I've made the last language file changes before the next update, so if you could update translate those as well, I can merge the pull request and include the translations in the new update.

**Pull request merged**